python/bifrost/psrdada.py

```diff
@@ -119,12 +123,12 @@ def __next__(self):
         else:
             del block
             self.reset()
-            raise EndOfDataStop('IpcBufBlock empty')
+            raise StopIteration()
```

Comment:
I thought there was some problem with `StopIteration` and that's why we moved to `EndOfDataStop`.  Maybe I'm thinking of something else?

Reply:
Ugh, this is a regression bug